mips_api/__init__.py

```diff
@@ -230,21 +419,31 @@ def _build_return(code, body):
 
             if event_path == api_routes['ApiChartOfAccounts']:
                 # return chart of accounts
-                return _build_return(200, mips_chart)
+                return _build_return(200, obj=mips_chart)
 
             elif event_path == api_routes['ApiValidTags']:
                 try:
-                    valid_tags = list_tags(params, mips_chart, omit_codes_list, extra_codes_dict)
+                    # build a list of valid tags
+                    valid_tags = list_tags(params, mips_chart)
+                    return _build_return(200, obj=valid_tags)
                 except Exception as exc:
-                    return _build_return(500, {"error": str(exc)})
+                    return _build_return(500, obj={"error": str(exc)})
 
-                # return valid tags
-                return _build_return(200, valid_tags)
+            elif event_path == api_routes['ApiCostCategoryRules']:
+                try:
+                    # scan account tags for account codes
+                    account_codes = collect_account_tag_codes(aws_tags)
+
+                    # build the rule snippet
+                    rule_snippet = list_rules(mips_chart, aws_tags, account_codes)
+                    return _build_return(200, obj=rule_snippet, ctype='yaml')
```

Comment:
I would return in json and let the user handle the conversion if needed.  The other option is to create a `format` parameter on the endpoint to allow the user to specify the return format (i.e. https://abc.org/rules?format=yaml)